src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -629,6 +628,104 @@ void EIRPlantLoopHeatPump::calcLoadSideHeatTransfer(EnergyPlusData &state, Real6
     // currentLoad will be met and there should? be some adjustment based on outlet water temp limit?
 }
 
+void HeatPumpAirToWater::calcLoadSideHeatTransfer(EnergyPlusData &state, Real64 const availableCapacity, Real64 const currentLoad)
```

Comment:
This may be out or scope for this new model but all plant equipment models should be passing back the load met (Real64 &currentLoad) to the plant manager (i.e., returned from the simulate function call to the plant manager) so the plant knows what the remaining load is. If the load is greater than the equipment capacity then the plant will have a remaining load to meet. All the simulate functions should look like this: `simulate(state, , , Real64 &curLoad) {}`